js/modules/k6/http/request_test.go

```diff
@@ -1306,6 +1306,10 @@ func TestRequestAndBatch(t *testing.T) {
 					name: "object no url key", code: `[ {method: "GET"} ]`,
 					expErr: `batch request 0 doesn't have a url key`, throw: true,
 				},
+				{
+					name: "multiple array as an argument", code: `["GET", "https://test.k6.io"],["GET", "https://test.k6.io"]`,
```

Comment:
```suggestion
					name: "multiple arguments", code: `["GET", "https://test.k6.io"],["GET", "https://test.k6.io"]`,
```
The validation applies to any type not only to arrays so better to relax the scope here

Reply:
Done. Updated the name.